coordinator/src/main/scala/com/socrata/datacoordinator/service/Main.scala

```diff
@@ -579,6 +579,12 @@ object Main extends DynamicPortMap {
         val secondaryManifestsResource = SecondaryManifestsResource(_: Option[String], secondaries,
           operations.datasetsInStore, common.internalNameFromDatasetId)
 
+        implicit val weightedCostOrdering: Ordering[Cost] = WeightedCostOrdering(
```

Comment:
I'm not sure it's worth pulling the cost ordering function out of the collocator. The weights in the config already give some levers to adjust so this seems like overkill. 
I think that it might be more valuable for code readability to keep the business logic of the collocation tightly coupled.

Reply:
I guess the main reason I was doing this is that I also use the ordering in Metric, and I didn't really want to pass the config values in to create the ordering in multiple places.